packages/web-react/src/components/Pill/README.md

```diff
@@ -0,0 +1,3 @@
+# Pill
```

Comment:
I've see that you copied out of `Tag`. Please look at `Alert`. That is much more better starting point. :-)